<?xml version="1.0"?>
<launch>

    <arg name="map_name" default="test"/>

    <node pkg="autopilot_map_server" type="rtk_mapping" name="rtk_mapping" output="screen">
        <param name="resolution" value="0.2"/>
        <param name="map_size" value="8192"/>
        <param name="region_of_interest" value="50.0"/>
        <param name="pointcloud_topic" value="/velodyne_points"/>
        <param name="projection" value="gauss"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz"
          args="-d $(find autopilot_map_server)/launch/rtk_mapping_rviz_cfg.rviz"/>

</launch>
